README.md

```diff
@@ -370,6 +524,46 @@ store.conventions.addDocumentInfoResolver({
 })();
 ```
 
+## Working with secured server
+
+1. Fill auth options object. Pass contents of the pem/pfx certificate, specify its type and (optionally) passphrase:
+```javascript
+const {DocumentStore, Certificate} = require('ravendb');
+
+const certificate = `
+-----BEGIN CERTIFICATE-----
+...
+-----END CERTIFICATE-----
+-----BEGIN RSA PRIVATE KEY-----
+...
+-----END RSA PRIVATE KEY-----
+`;
+
+let authOptions = {
+  certificate: certificate,
+  type: Certificate.Pem,
+  password: 'my passphrase' // optional  
+};
+``` 
+
+_Note_. Pfx certificates content should be passed as `Buffer` object.
```

Comment:
For PEM can we just detect whether user passed a Buffer or a string and act accordingly?

Reply:
Also since we support PEM as well as PFX let's split the above example into 2 bullet points: for PFX and PEM and show how to use either of those.

Reply:
@gregolsky you can pass pem also as an Buffer 
For example, if you have .pem file you can just do fs.readFileSync and pass result to auth options without calling .toString()
Pfx can be passed only as Buffer as it's an binary file and it destroys while converting to string